### DISCUSSION TITLE
Select

### DISCUSSION THREAD

**Discussion by jaallaall** [mantinedev/mantine#7128] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.12

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

None

### Describe the bug

Hello, I have a problem with this component
For example, when for the component
 Key.form
 and
 onchange
 I define
 When I search and send a request, I get the results on the server side
 It is not possible to choose
 It means it is selected but not displayed
 When I remove one of the above it works correctly
 what's the problem

### Self-service

- [X] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
The question is not clear, provide a sandbox with a minimal reproduction

**Reply by jaallaall** [reply]
Hello
My question is related to this component
That's why I'm posting it here
Please see this code
Is it implemented correctly
I implemented this component based on your source to create a new option
But it seems incomplete and has some problems
Can you please check it and tell me the problems

https://codesandbox.io/p/sandbox/blissful-shadow-tfdph4?file=%2Fsrc%2FApp.js
